### PR TITLE
Upgraded the lodash version to fix Prototype Pollution vulnerability (CVE-2019-10744) 

### PR DESCRIPTION
**Pull request by apoorvmalhotra**
Upgraded the lodash version to fix Prototype Pollution vulnerability (CVE-2019-10744)
Please refer to https://app.snyk.io/vuln/SNYK-JS-LODASH-450202

### CODE REVIEW

**Comment by apoorvmalhotra** [comment]
@AndrewKeig have time for a quick review?

**Comment by colthreepv** [comment]
It LGTM for me, this library does not have any invocation of `defaultsDeep`, the function reported to achieve prototype pollution.

If tests run I would give a green light

**Comment by apoorvmalhotra** [comment]
Thanks @colthreepv. This has been flagged as one of the security vulnerabilities in my project as I use the `express-validation` library. Can you please publish a new version as well with this change?

**Comment by apoorvmalhotra** [comment]
@colthreepv @AndrewKeig can you please merge this PR and create a new package?

**Comment by isaachinman** [comment]
@apoorvmalhotra First, the caret means you can install any minor/patch off v4 already. Second, this PR has conflicts and will not be considered until you resolve them.

File: package.json [hunk 1]
```diff
@@ -15,8 +15,8 @@
     "url": "https://github.com/andrewkeig/express-validation"
   },
   "dependencies": {
-    "@types/joi": "^13.0.5",
-    "lodash": "^4.9.0"
+    "lodash": "^4.17.15",
```
Comment (isaachinman) [inline package.json]: Why did these reverse order and become un-alphabetised?

Comment (apoorvmalhotra) [reply package.json]: @isaachinman  I have fixed the order of lodash.

**Pull request merged**